website/src/pages/ProjectCard/card.css

```diff
@@ -26,14 +28,17 @@
   height: 100%;
 }
 .card_button {
+  position: relative;
+  height: 20px;
   padding: 2%;
+  background-color: white;
+  margin-left: -350px;
```

Comment:
In general, we would prefer using `rem`/`em` units in most of our css for better responsiveness and use `px` unit only if a necessity. Also, consider removing `-350px` margin, because explicit margin might break in other screen sizes. Thank you

Reply:
ok

Reply:
> Hi @SanjaySinghRajpoot , seems like the Roadmap Page UI breaks on Dark mode. Can you please consider fixing this part? ![image](https://user-images.githubusercontent.com/47717492/145587932-b623fde6-8a4f-42cf-8dfa-d4c706a6fa9d.png)

I have fixed this bug and added required margin to the button in  `rem` format instead of `pixel`. Please review @sohamsshah 

Thank You.